src/data/themes/index.ts

```diff
@@ -1,22 +1,35 @@
-import type { ImageMetadata } from '@astrojs/image'
+import type { ImageMetadata } from '@astrojs/image/dist/vite-plugin-astro-image.js'
 import data from '../themes.json'
 import type { Link, Theme } from '../../types.js'
 
 interface ThemeData {
     title: string
     description: string
-    image: { src: string; alt: string }
+    fullDescription?: string
+    image: { src: string; alt: string; }
+    images?: { src: string; alt: string; }[]
     repoUrl: Link
     npmUrl?: Link
     demoUrl?: Link
+    links?: Link[]
     categories: string[]
     featured?: number
     slug: string
     stars: number
+    tags?: string[]
 }
 
-const allImages: { [key: string]: () => Promise<{ default: ImageMetadata }> } =
-    import.meta.glob('./images/*.{png,jpg,jpeg}')
+const allImages = import.meta.glob('./images/*.{png,jpg,jpeg}') as { [key: string]: () => Promise<{ default: ImageMetadata }> }
+
+async function resolveImage(src: string) {
+    if (!(src in allImages)) {
+        throw new Error(`[loadThemes] "${src}" image not found! Does it exist in /src/data/themes/images?`)
+    }
+
+    const mod = await allImages[src]()
+
+    return mod.default
+}
```

Comment:
The PR will use image URLs that are uploaded to Netlify from the form. Will that cause an error here?

Reply:
Can the form submission webhook commit images as part of the PR?  I'm not sure if there's any guarantee how long the images are available on Netlify's CDN, especially if we delete the form submission once we merge in each PR